src/content/en/2019/page-weight.md

```diff
@@ -6,102 +6,84 @@ description: Page Weight chapter of the 2019 Web Almanac covering why page weigh
 
-## More bandwidth isn’t a magic bullet for web performance
+Here's a quick glossary of the page composition metrics the HTTP Archive tracks, and how much they matter in terms of performance and user experience:
 
-Even if more people had access to better devices and cheaper connections, that wouldn’t be a complete solution. Double the bandwidth doesn't equal twice as fast. This assumption has had holes poked in it a number of times ([such as this example](https://developer.akamai.com/blog/2015/06/09/heres-why-more-bandwidth-isnt-magic-bullet-web-performance), which demonstrated that increasing bandwidth by up to 1233% made pages just 55% faster). 
+- **Total size** - This is the total weight in kilobytes of the page. It matters especially to mobile users who have limited and/or metered data.
 
-The problem is latency. Most of our networking protocols require a lot of round-trips. Each of those round trips imposes a latency penalty. Latency is governed, at the end of the day, by the speed of light. For as long as latency continues to be a performance problem (which is to say, for the foreseeable future), the major performance culprit will continue to be the fact that a typical web page today contains a hundred or so assets hosted on dozens of different servers. Many of these assets are unoptimized, unmeasured, unmonitored – and therefore unpredictable.
+- **HTML** - HTML is typically the smallest resource on the page. Its performance risk is negligible.
 
+- **Images** - Unoptimized images are often the greatest contributor to page bloat. Looking at the 90th percentile of data gathered for the Almanac, images accounted for a whopping 5220KB of a roughly 7MB page. In other words, images comprised almost 75% of the total page weight. And if that already wasn't enough, the number of images on a page has been linked to lower conversion rates on retail sites. (More on that later.)
 
-## What types of assets does the HTTP Archive track, and how much do they matter?
+- **JavaScript** - JavaScript matters. A page can have a relatively low JS weight but still suffer from JS-inflicted performance problems. Even a single 100KB third-party script can wreak havoc with your page. The more scripts on your page, the greater the risk.
 
-Here’s a quick glossary of the page composition metrics the HTTP Archive tracks, and how much they matter in terms of performance and user experience:
+  It's not enough to focus solely on blocking JS. It's possible for your pages to contain zero blocking resources and still have less-than-optimal performance because of how your JavaScript is rendered. That's why it's so important to understand CPU usage on your pages—because JavaScript consumes more CPU than all other browser activities combined. While JS blocks the CPU, the browser can't respond to user input. This creates what's commonly called "jank"—that annoying feeling of jittery, unstable page rendering.
```

Comment:
```suggestion
  It's not enough to focus solely on blocking JavaScript. It's possible for your pages to contain zero blocking resources and still have less-than-optimal performance because of how your JavaScript is rendered. That's why it's so important to understand CPU usage on your pages, because JavaScript consumes more CPU than all other browser activities combined. While JavaScript blocks the CPU, the browser can't respond to user input. This creates what's commonly called "jank": that annoying feeling of jittery, unstable page rendering.
```